extensions/ql-vscode/src/authentication.ts

```diff
@@ -21,30 +21,24 @@ export class Credentials {
   static async initialize(context: vscode.ExtensionContext): Promise<Credentials> {
     const c = new Credentials();
     c.registerListeners(context);
-    await c.initializeOctokit(false);
+    await c.createOctokit(false);
     return c;
   }
 
-  private async initializeOctokit(createIfNone: boolean) {
-    // If `createIfNone` is true, a dialog pops up asking the user to authenticate as soon as the extension starts.
-    // Initializing with `createIfNone: false` for now, so we can have a more quiet prompt, i.e. a numbered label on
-    // the "accounts" icon in the activity bar.
+  private async createOctokit(createIfNone: boolean): Promise<Octokit.Octokit | undefined> {
     const session = await vscode.authentication.getSession(GITHUB_AUTH_PROVIDER_ID, SCOPES, { createIfNone: createIfNone });
```

Comment:
Nit: This can be simplified:

```suggestion
    const session = await vscode.authentication.getSession(GITHUB_AUTH_PROVIDER_ID, SCOPES, { createIfNone });
```